src/Lender.sol

```diff
@@ -160,9 +157,9 @@ contract Lender {
     /// @param a the APWine contract
     constructor(address s, address p, address a) {
         admin = msg.sender;
-        swivelAddr = s;
-        pendleAddr = p;
-        apwineAddr = a;
+        protocolAddressess.push(s);
```

Comment:
Semi-related to my previous question's train of thought --

It looks like these are just pushing to the array so would be index 0,1,2.

So with regard to the comment on line 38 

> @dev these addresses are references by an implied enum; adapters hardcode the index for their protocol

Is this a different enum than the protocols enum? Or is the thought to have them keep the same index per protocol across both `protocols` and `protocolAddresses`

Reply:
Yes, this is different from the protocols enum. This array is purely meant for calling into certain protocol addresses. For example, in Swivel's lend call, we need to call `ISwivel(swivelAddr).initiate`. This array stores that address. It also allows us to modularly support any new protocols that have special addresses.